.github/workflows/pytest.yml

```diff
@@ -35,6 +35,12 @@ jobs:
           python -m pip install --upgrade pip -r requirements-dev.txt
           pip install -e .
 
+      - name: Use latest version of git wrt. the Ubuntu image in use
+        run: |
+          sudo apt remove git git-man
+          sudo add-apt-repository --remove ppa:git-core/ppa
+          sudo apt install git
```

Comment:
I am not knowledgable enough to judge this, but isn't there an `apt update` missing before the installation? Or is it intentionally missing?

Reply:
`apt update` is needed when you add a new source and need to update package information. When you remove a source, there is no new info to get :)